pom.xml

```diff
@@ -64,6 +64,35 @@
             <artifactId>ioutils</artifactId>
             <version>047e401d73</version>
         </dependency>
+
+        <!-- Dependencies for Diffblue Deeptest -->
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>1.6.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito-common</artifactId>
```

Comment:
`powermock-api-mockito-common` is not neccessary as it will be brought in by `powermock-api-mockito` anyway.